lifemonitor/api/models/services/travis.py

```diff
@@ -85,6 +86,7 @@ def _build_url(self, path, params=None):
         query = "?" + urllib.parse.urlencode(params) if params else ""
         return urllib.parse.urljoin(self.api_base_url, path + query)
 
+    @cache.memoize(10)
```

Comment:
Can we replace this hardwired number with a self-documenting variable?

Reply:
fixed in https://github.com/crs4/life_monitor/pull/145/commits/60dd10bcb0cac0bf248bbddb60cc267f91016b34